test/system/src/test/java/io/pravega/test/system/MultiReaderWriterWithFailOverTest.java

```diff
@@ -309,12 +303,19 @@ public void multiReaderWriterWithFailOverTest() throws Exception {
         log.info("Test {} succeeds ", "MultiReaderWriterWithFailOver");
     }
 
+    private void cleanUp() throws Exception {
```

Comment:
dont you want to catch exceptions while cleaning up and react to them accordingly.